pkgs/by-name/xw/xwin/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  owner = "Jake-Shadle";
+  repo = "xwin";
+in
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
This kinda lambdas are now supported by the Rust builder and thus are preferred over rec